tests/Unit/Css/StyleRuleTest.php

```diff
@@ -0,0 +1,182 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pelago\Emogrifier\Tests\Unit\Css;
+
+use Pelago\Emogrifier\Css\StyleRule;
+use PHPUnit\Framework\TestCase;
+use Sabberworm\CSS\Rule\Rule;
+use Sabberworm\CSS\RuleSet\DeclarationBlock;
+
+/**
+ * @covers \Pelago\Emogrifier\Css\StyleRule
+ */
+final class StyleRuleTest extends TestCase
+{
+    /**
+     * @return array<string, array{0: array<int, string>, 1: string}>
+     */
+    public function provideSelectors(): array
+    {
+        return [
+            'single selector' => [['h1'], 'h1'],
+            'two selectors' => [['h1', 'p'], 'h1, p'],
+        ];
+    }
+
+    /**
+     * @test
+     *
+     * @param array<int, string> $selectors
+     *
+     * @dataProvider provideSelectors
+     */
+    public function getSelectorsReturnsSelectorsProvidedInConstructor(array $selectors): void
```

Comment:
`To` might be preferable to `In` in the method name.  There is a similarly named method that needs changing (see below); the naming for each should be consistent.